src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -4262,7 +4262,7 @@ void MacroAssembler::lookup_interface_method(Register recv_klass,
     lea(recv_klass, Address(recv_klass, itable_index, Address::times_ptr, itentry_off));
   }
 
-  // for (scan = klass->itable(); scan->interface() != NULL; scan += scan_step) {
+  // for (scan = klass->itable(); scan->interface() != null; scan += scan_step) {
```

Comment:
nullptr

Reply:
Missed change to nullptr as you suggested.